packages/zowe-explorer/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 - Fixed an issue where a TypeError occurred when applying VS Code proxy settings to an invalid session. [#3425](https://github.com/zowe/zowe-explorer-vscode/issues/3425)
 - Fixed issue where the 'Delete' key binding for the USS tree returns a 'contextValue' error. [#2796](https://github.com/zowe/zowe-explorer-vscode/issues/2796)
+- Fixed the issue where user is unable to open a renamed sequential dataset from the DS tree. [#3345](https://github.com/zowe/zowe-explorer-vscode/issues/3345)
```

Comment:
Some minor edits, please: "Fixed **an** issue ... sequential **data set** from the **Data Sets tree view**."